### DISCUSSION TITLE
Unable to Get GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by Kunal301** [community/community#49924] [GitHub Education]
I have completed my profile on Github (@Kunal301) and also I am providing the ID card of my college which clearly determines that I am a student of that college also all the dates are mentioned on that ID Card.
I have done everything still my request is every time canceled saying that there are some common verification problems. Now I want to know what are those problems and they say that completed my profile I have also done that still my request is canceled more than 3 times. Please anyone reply with what can I do to verify myself as a student.

**Comment by Ntshangase** [comment]
Here is a resource that might be helpful: https://github.com/community/community/discussions/17814
